lib/plivo/resources/multipartycalls.rb

```diff
@@ -74,14 +76,19 @@ def add_participant(role,
         valid_param?(:to, to, String, false ) unless to.nil?
         valid_multiple_destination_nos?(:to, to, role: role, delimiter: '<', agent_limit: 20) unless to.nil?
         valid_param?(:call_uuid, call_uuid, String, false ) unless call_uuid.nil?
+        valid_param?(:caller_name, caller_name, String, false) unless caller_name.nil?
+        valid_range?(:caller_name, caller_name.length, false, 0, 50) unless caller_name.nil?
         valid_url?(:call_status_callback_url, call_status_callback_url, false) unless call_status_callback_url.nil?
         valid_param?(:call_status_callback_method, call_status_callback_method.upcase, String, false, %w[GET POST])
         valid_param?(:sip_headers, sip_headers, String, false) unless sip_headers.nil?
         valid_param?(:confirm_key, confirm_key, String, false , %w[0 1 2 3 4 5 6 7 8 9 # *]) unless confirm_key.nil?
         valid_url?(:confirm_key_sound_url, confirm_key_sound_url, false) unless confirm_key_sound_url.nil?
         valid_param?(:confirm_key_sound_method, confirm_key_sound_method.upcase, String, false, %w[GET POST])
         is_one_among_string_url?(:dial_music, dial_music, false, %w[real none])
-        valid_range?(:ring_timeout, ring_timeout, false, 15, 120)
+        valid_param?(:ring_timeout, ring_timeout, [String,Integer], false)
```

Comment:
@SaiTharunPlivo, are we accepting string params for ring_timeout and delay_dial? Also, please add a validation by splitting the string with **<** delimiter and a check if each of the passed values fall within the respective bounds.

Reply:
@Koushik-Ayila  yes when there are multiple destinations to be dialled, then ring_timeout and delay_dial can be given as multilple values separated by '<' in a string. The validation is done in the function 'valid_multiple_destination_integers?'